### ISSUE TITLE
Training loss is negative.

### ISSUE THREAD

**Issue by guangkaixu** [baegwangbin/surface_normal_uncertainty#7]
Hi, thanks so much for your excellent work on the surface normal estimation. When I was training on the taskonomy dataset with the paper-proposed loss function 'UG_NLL_ours' (i.e., simply replace the img and norm paths of the Nyuloader), the loss tended to decrease and even be negative. Is it okay for training? Or there may exist some mistakes.

**Comment by baegwangbin** [comment]
Hi, yes this is very natural. Probability density, unlike probability mass, can be higher than 1, which would make the loss negative.

**Comment by guangkaixu** [comment]
Oh, I understand. Thank you very much for the pretty work and fast response!
